### ISSUE TITLE
Example Jupyter Notebook for TGN 

### ISSUE THREAD

**Issue by shenyangHuang** [fpour/DG_process#1]
Create an example Jupyter notebook which will contain the following steps
1). download dataset from dgb, say canVote
2). process dataset into TGN readable format
3). define TGN architecture 
4). train TGN on the training set while using validation set for evaluation
5). use our provided evaluator to evaluate test set performance with a). random  b). historical  c). inductive negative sampling strategies
6). report the above evaluation results

Notes: 
* don't forget to fix random seed for TGN training, should be 100% reproducible 
*  also check that dataset split is fixed and using the default split
* provide detailed instruction on dataset processing and evaluator usage

**Comment by abubakardaud** [comment]
https://colab.research.google.com/drive/1mPcWst_gkPBQ1wS7JFXR3ynXTjPXpq7U?usp=sharing
This has been completed here. 

For 
1 2 3 4. 

5,6 need to be done

**Comment by abubakardaud** [comment]
https://colab.research.google.com/drive/1SmTfghdoWC37-Aucd7N3IOhukWZn9oOO?usp=sharing 
EdgeBank working

**Comment by shenyangHuang** [comment]
@fpour  Can you check if the run time for EdgeBank is correct here?
